wire-ios-sync-engine/Tests/Source/SessionManager/SessionManagerTests.swift

```diff
@@ -470,6 +470,23 @@ class SessionManagertests_AccountDeletion: IntegrationTest {
         XCTAssertFalse(FileManager.default.fileExists(atPath: accountFolder.path))
     }
 
+    func testThatItDeletesTheLastEventID_WhenDeletingActiveUserSessionAccount() throws {
+        // given
+        XCTAssert(login())
+
+        let account = sessionManager!.accountManager.selectedAccount!
```

Comment:
guard .... else {
  XCTFail()
}

Reply:
fixed in c09642d602e1ab8156a0868fa601ad6d4a53a861